CONTRIBUTING.rst

```diff
@@ -3,24 +3,20 @@ Contributing guide
 
 Thanks for your interest in contributing to cpplint.
 
-Any kinds of contributions are welcome: Bug reports, Documentation, Patches.
+Any kinds of contributions are welcome: Bug reports, Documentation, Patches. If you need some ideas, you may check out some of the tasks below:
 
-However, cpplint is a bit special as a project because it aims to closely follow what Google does in the upstream repository.
-That means Google remains the source of all major requirements and functionality of cpplint whereas this fork adds extensions to cpplint run on more environments and in more companies.
-The difference between this cpplint and Google should remain so small that anyone can at a glance see there is no added change that could be regarded as a security vulnerability.
+* Reducing interference with other popular tools
+* Update cpplint for modern python versions
+  * Converting setup.py to a pyproject.toml
+* Porting cpplint to other environments
+  * Making tests, especially clitest, work on Windows
```

Comment:
https://github.com/cpplint/cpplint/pull/247 fixes this.

IMO this file is not the place to put any specific problems.  Issues should be created for any problems, and those issues can be pinned to the top of the issue list if they are especially important.

Reply:
Well, just having a list of things you shouldn’t do seems a bit too negative to me, so I think we should put some more positive stuff to balance it out. Thoughts?

Reply:
Be positive about telling potential contributors going and reading the existing issues that have been raised...?